src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiration-notification/pwd-expiration-notification.component.ts

```diff
@@ -37,11 +37,9 @@ export class PwdExpirationNotificationComponent implements OnInit {
   }
 
   private getExpirationDays(pwdExpirationDate: number): number {
-    if (pwdExpirationDate) {
-      const current = new Date();
-      const expiration = new Date(pwdExpirationDate * 1000);
-      return Math.floor((expiration.valueOf() - current.valueOf()) / (1000 * 3600 * 24));
-    }
+    const current = new Date();
+    const expiration = new Date(pwdExpirationDate * 1000);
+    return Math.floor((expiration.valueOf() - current.valueOf()) / (1000 * 3600 * 24));
```

Comment:
If I revert this I will need to add `return undefined;` at the end of the function.
I though this was a better fix.

Reply:
If `pwdExpirationData` is not set you will call `Date` with `NaN` this would return 'NaN' at the end. I'm not sure if it will break something but it has much more potential to do so.

Reply:
I tested it locally doesn't seem to break anything.